charts/base-cluster/README.md.gotmpl

```diff
@@ -149,3 +149,7 @@ Please check the following (urgent) upgrade notes before upgrading:
 [k8s upgrade notes v1.27](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.27.md#urgent-upgrade-notes)
 
 [redis 7.0 release notes](https://raw.githubusercontent.com/redis/redis/7.0/00-RELEASENOTES) before upgrading.
+
+### To 39.0.1
+
+From 39.0.1 onwards, Prometheus will send alerts for levels where human interference might be necessary in regards to Node CPU / Memory usage as well as HikariCP.
```

Comment:
That this is onwards this version is already stated by the version mentioned above

Reply:
Changed